user_account/api/urls.py

```diff
@@ -1,13 +1,9 @@
-from django.urls import path
+from django.urls import include, path
 from rest_framework_simplejwt import views as jwt_views
 
-from user_account.api.views import UserCreateView, UserDetailView
-
 app_name = "user_account"
 
 urlpatterns = [
-    path("register/", UserCreateView.as_view(), name="register"),
-    path("token/", jwt_views.TokenObtainPairView.as_view(), name="token_obtain_pair"),
-    path("token/refresh/", jwt_views.TokenRefreshView.as_view(), name="token_refresh"),
-    path("user/<int:pk>", UserDetailView.as_view(), name="user_detail"),
+    path("auth/", include("djoser.urls")),
```

Comment:
it's work ?
How Django resolve 2 the same url ?

Reply:
In this case it mixes them and gives you an access to endpoints from both of them. If you remove "djoser.urls", you lose access to account seccion including "auth/users/me" which returns your user email, and if you remove "djoser.urls.jwt" you lose access to jwt-authentication.